pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -485,13 +485,18 @@ class Build {
         def parallel = 'None'
         def num_machines = '1'
         def remoteTargets = [:]
+        def additionalLabel = ''
+        if (jdkVersion >= 19 && platform == 'ppc64_aix') {
+                // Jck19+ AIX must run on AIX 7.2
+                additionalLabel = 'sw.os.aix.7_2'
+        }
```

Comment:
See my comment [[remoteTriggerJckTests for jdk19+ AIX needs to use LABEL=sw.os.aix.7_2](https://github.com/adoptium/ci-jenkins-pipelines/issues/521#top)](https://github.com/adoptium/ci-jenkins-pipelines/issues/521#issuecomment-1334404867). @sxa could https://ci.adoptopenjdk.net/label/aix720&&ci.role.test/ be updated with label `[sw.os.aix.7_2]`?

Reply:
This way we can just do `context.MapParameter(name: 'LABEL_ADDITION', value: buildConfig.ADDITIONAL_TEST_LABEL)]),`

Reply:
yes, it would be nice to have a consistent labelling, that maybe quite a large piece of work

Reply:
Approved. Might be worth to open an issue to track the update aix72 label for testing.

Reply:
> @sxa could https://ci.adoptopenjdk.net/label/aix720&&ci.role.test/ be updated with label sw.os.aix.7_2?

Done. I believe @andrew-m-leonard has the access to make those changes too.

Reply:
Thanks @sxa @sophia-guo i'm using your suggestion, thank you.